### PR TITLE
Bookmark button

### PR DESCRIPTION
**Pull request by ThysBat**
I added the javascript to toggle the bookmark button on a question card. Please have a look! Thanks 😊

### CODE REVIEW

File: css/pages/bookmarks.css
Comment (FelineHuhn) [inline css/pages/bookmarks.css:1]: If this is an empty file, you should delete it.

File: js/bookmarks.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const bookmarks = document.querySelectorAll('[data-js="bookmark"]');
+
+console.log(bookmarks);
```
Comment (FelineHuhn) [inline js/bookmarks.js:3]: You always should remove console.log's for a Pull Request :)

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const bookmarks = document.querySelectorAll('[data-js="bookmark"]');
+
+console.log(bookmarks);
+
+for (const bookmark of bookmarks) {
```
Comment (FelineHuhn) [inline js/index.js:5]: Loops are actually next week :D

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const bookmarks = document.querySelectorAll('[data-js="bookmark"]');
+
+console.log(bookmarks);
```
Comment (FelineHuhn) [inline js/index.js:3]: Same here (see above)

File: pages/bookmarks.html [hunk 4]
```diff
@@ -5,7 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Recap Project 1 - Quiz App</title>
-    <link rel="stylesheet" href="../css/pages/bookmark.css" />
+    <link rel="stylesheet" href="../css/pages/bookmarks.css" />
```
Comment (FelineHuhn) [inline pages/bookmarks.html:8]: You could also link to styles.css, and create the bookmark-styles as components and not as a new stylesheet for the whole bookmark page.

**Review by FelineHuhn — APPROVED** [review]
Nice Work David! 🚀 I just left ja some tiny comments below. Keep going 🙌🏼

**Comment by ThysBat** [comment]
Thanks for the review, I appreciate the comments!

**Pull request merged**